spec/support/acceptance/litmus_helpers.rb

```diff
@@ -0,0 +1,74 @@
+require 'puppet_litmus'
+
+class LitmusTarget
+  include PuppetLitmus
+
+  attr_reader :uri
+
+  def initialize(uri)
+    @uri = uri
+  end
+end
+
+module LitmusHelpers
+  extend PuppetLitmus
+
+  def self.instance
+    target_host = ENV['TARGET_HOST']
+    unless target_host
+      raise "LitmusHelpers.instance can only be called when running the tests (when ENV['TARGET_HOST'] is set)"
+    end
+    LitmusTarget.new(ENV['TARGET_HOST'])
+  end
+end
+
+# These helpers are a useful way for tests to reuse
+# Litmus' helpers when they want to do stuff on nodes
+# that may not be the current target host (like e.g.
+# the master or the ServiceNow instance).
```

Comment:
Example of when master isn't target host is when we have an agent, and the agent attempts to modify master's site.pp to do stuff.